dev-tools/generate_notice.py

```diff
@@ -33,7 +35,16 @@ def get_library_path(license):
     return "/".join(split)
 
 
-def add_licenses(f, vendor_dirs):
+def read_versions(vendor):
+    libs = []
+    with open(os.path.join(vendor, "vendor.json")) as f:
```

Comment:
It adds dependencies on `govendor` but I think that is fine for now as I'm still quite happy with govendor.